support/source/ios/Mockifer.xcodeproj/project.pbxproj

```diff
@@ -316,7 +322,7 @@
 				GCC_WARN_UNUSED_VARIABLE = YES;
 				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
 				MTL_ENABLE_DEBUG_INFO = YES;
-				ONLY_ACTIVE_ARCH = NO;
+				ONLY_ACTIVE_ARCH = YES;
```

Comment:
Only building the active architecture is not desirable as the iOS library output should be able to run on all architectures. Suggest reverting the project.pbxproj file from your PR.